docs(srv): split troubleshootings into multiple pages MTA-5198 (#3955)
scaleway/docs-content: serverless/functions/troubleshooting/cannot-create-namespace.mdx

---
meta:

>   title: I am unable to create a Serverless Functions namespace
>   description: Troubleshoot problems encountered while creating namespaces for Scaleway Serverless Functions.
> content:
>   h1: My function fails after trying too many times

Reviewer: ```suggestion
  title: I am unable to create a Serverless Functions namespace
  description: Troubleshoot problems encountered while creating namespaces for Scaleway Serverless Functions.
content:
  h1: I am unable to create a Serverless Functions namespace
```

  paragraph: Troubleshoot problems encountered while creating namespaces for Scaleway Serverless Functions.
tags: serverless functions troubleshooting issue error namespace creation unable
dates:
  validation: 2024-11-06
  posted: 2024-11-06
categories:
  - serverless
---

## Problem

I cannot create a Serverless Functions namespace.

### Cause

This issue can happen for the following reasons:

> - You have created too many namespaces and reached [your account's quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#serverless-functions).

Reviewer 2: ```suggestion
- You have created too many Serverless Funtcions namespaces and reached [your account's quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#serverless-functions).
```

> - You created too many namespaces and reached [your account's quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#serverless-functions).
> 
> - You created too many Registry namespaces and reached [your account's quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#container-registry).

Reviewer: ```suggestion
- You have created too many namespaces and reached [your account's quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#serverless-functions).

- You have created too many Registry namespaces and reached [your account's quota](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/#container-registry).
```

### Possible solutions

> Make sure that you did not exceed the maximum number of allowed namespaces with the [Scaleway console](https://console.scaleway.com/) or with the [API](https://www.scaleway.com/en/developers/api/serverless-functions/#path-namespaces-list-all-your-namespaces).

Reviewer: ```suggestion
Make sure that you have not exceeded the maximum number of allowed namespaces with the [Scaleway console](https://console.scaleway.com/) or with the [API](https://www.scaleway.com/en/developers/api/serverless-functions/#path-namespaces-list-all-your-namespaces).
```
